README.md

```diff
@@ -33,7 +26,8 @@ detectors             | detector-specific setups
 ```
 git clone https://github.com/ess-dmsc/essdaq.git
 cd essdaq
-./install.sh
+./install.sh          # for Ubuntu
+./install)centos7.sh  # for CentOS 
 ```
 
```

Comment:
And I guess there could be some more info explaining the new auto/manual parameters for the install scripts? In general, this readme could use an update.

Reply:
Readme updated with more explanations.